services/x-files.js

```diff
@@ -1,4 +1,4 @@
-export const fetchgithub = (username) => {
+export const fetchCharacters = (username) => {
```

Comment:
you don't need username in the parens if you're just getting all characters. you will need a characterId in there when you make your fetch function for the detail page.